File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumCornerTypeValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;
import javax.annotation.Nonnull;

/**
 *  
 A writer that can write a value as a type of a corner.
 

 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumCornerTypeValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    *  
    Writes the value expressed as a corner type.
    
    

    * @param value The type of a corner.
    */
    void writeCornerType(@Nonnull CesiumCornerType value);
}

File: DotNet/CesiumLanguageWriter/Advanced/ICesiumCornerTypeValuePropertyWriter.cs
namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a type of a corner.
    /// </summary>
    public interface ICesiumCornerTypeValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value expressed as a corner type.
        /// </summary>
        /// <param name="value">The type of a corner.</param>
        void WriteCornerType(CesiumCornerType value);
    }
}